File: markov/src/python/saveToJSON.py
import json

# マルコフのチェーンデータ
chain = {}

# 保存先のファイルパス
file_path = "/path/to/save.json"

# JSONファイルに保存
with open(file_path, "w") as json_file:
    json.dump(chain, json_file)

File: markov/src/ts/saveToJSON.ts
import * as fs from 'fs';

// マルコフのチェインデータ
const chain: Record<string, any> = {};

// 保存先のファイルパス
const file_path = "/path/to/save.json";

// JSONファイルに保存
fs.writeFileSync(file_path, JSON.stringify(chain));
